translation-config.yml

```diff
@@ -1,6 +1,46 @@
+locales:
+  default: en_US
+  supported:
+    - ar
+    - de
+    - en_GB
+    - es
+    - fr_CA
+    - fr_FR
+    - it
+    - ja
+    - nl
+    - pl
+    - pt
+    - ru
+    - sv
+    - zh_CN
+    - zh_HK
+
 bundles:
   - path: .
     extension: properties
     default_locale: en_US
   - path: src/main/resources
     extension: json
+
+snapshot:
+  copy:
+    - en_GB
+
+io:
```

Comment:
yml should be a little bit more readable. Can we get rid of this nested yml list and just do a top level `generator` and `importer` instead?

Reply:
It could, I'll consider it :)